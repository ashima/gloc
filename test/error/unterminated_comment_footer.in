


/* This is an unterminated comment in the footer of an empty shader
