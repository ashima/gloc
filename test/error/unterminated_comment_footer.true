File 0, line 4, col 1:
unterminated comment

