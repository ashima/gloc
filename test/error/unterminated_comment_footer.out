File 0, line 5, col 1:
unterminated comment

